da/dispersal.py

```diff
@@ -69,9 +76,18 @@ def _build_attestation_message(encoded_data: EncodedData) -> bytes:
     def disperse(self, encoded_data: EncodedData) -> Optional[Certificate]:
         attestations = []
         attested_message = self._build_attestation_message(encoded_data)
-        for node, pk, blob in zip(self.settings.nodes_ids, self.settings.nodes_pubkey,  self._prepare_data(encoded_data)):
+        signed = Bitfield(False for _ in range(len(self.settings.nodes_ids)))
+        blob_data = zip(
+            range(len(self.settings.nodes_ids)),
+            self.settings.nodes_ids,
+            self.settings.nodes_pubkey,
+            self._prepare_data(encoded_data)
+        )
```

Comment:
Do you think it's worth sorting `settings.nodes_ids` so that we are sure that everyone is mapping each node id to the same index.

Reply:
Yeah, that could be useful. But I have been thinking actually it may go further than that:
My initial idea was that ordering/participation_set was coming from the on-chain snapshot of some specific epoch. But I am not really sure it can be done like that, or if it actually should. Otherwise, participants that attested DA may be gone by the time the data is included in a block. In that case we would need some extra validation process (?)
@megonen @alvatar 

PD: I'll order it for now.